pkg/alertmanager/amcfg.go

```diff
@@ -2766,6 +2773,35 @@ func (rc *rocketChatConfig) sanitize(amVersion semver.Version, logger *slog.Logg
 	return rc.HTTPConfig.sanitize(amVersion, logger)
 }
 
+func (ic *incidentioConfig) sanitize(amVersion semver.Version, logger *slog.Logger) error {
```

Comment:
can we validate URLs too?

Reply:
hi could you pls clarify, since for this config I added both URL and URL file and have the check for enforcing one's configured, following https://github.com/prometheus/alertmanager/blob/main/config/notifiers.go#L607.